features/facebookImport/src/model/analyses/off-facebook-events-analysis.js

```diff
@@ -1,17 +1,36 @@
+import React from "react";
+
 import RootAnalysis from "./root-analysis.js";
+import i18n from "../../i18n.js";
+import { groupOffFacebookEventsByType } from "./utils/off-facebook-events-utils.js";
+
+import OffFacebookMiniStory from "../../components/offFacebookMinistory/offFacebookMinistory.jsx";
 
 export default class OffFacebookEventsAnalysis extends RootAnalysis {
     get title() {
-        return "Off-Facebook Activity";
+        return i18n.t("offFacebookEventsMiniStory:title");
     }
 
     async analyze({ facebookAccount }) {
```

Comment:
I added initially some `isActive` checks, but most could be removed.
For example

```
this._companiesCount = facebookAccount.offFacebookCompaniesCount;
this._purchasesCount = <...>;
this.active = this._companiesCount.length > 0;
```

Reply:
What do you mean by that exactly? Should I remove that?

Reply:
We do not need the if, and can also directly initialize `this._purchasesCount` with the result of `groupOffFacebookEventsByType `, as that would anyway return 0 if there are no events. It's all optional but would simplify the method